docs/README.rst

```diff
@@ -1,41 +1,170 @@
+            f"running in pid {os.getpid()}"
+        )
+
+       await trio.sleep_forever()
+
+
+    async def main():
+
+        async with tractor.open_nursery() as n:
+
+            for i in range(cpu_count()):
+                await n.run_in_actor(target, name=f'worker_{i}')
+
+            print('This process tree will self-destruct in 1 sec...')
+            await trio.sleep(1)
+
+            # you could have done this yourself
+            raise Exception('Self Destructed')
+
+
+    if __name__ == '__main__':
+        try:
+            trio.run(main)
+        except Exception:
+            print('Zombies Contained')
+
+
+If you can create zombie child processes (without using a system signal)
+it **is a bug**.
+
+
+"Native" multi-process debugging
+--------------------------------
+Using the magic of `pdb++`_ and our internal IPC, we've
+been able to create a native feeling debugging experience for
+any (sub)-process in your ``tractor`` tree.
+
+.. code:: python
+
+    from os import getpid
+
+    import tractor
+    import trio
+
+
+    async def breakpoint_forever():
+        "Indefinitely re-enter debugger in child actor."
+        while True:
+            yield 'yo'
+            await tractor.breakpoint()
 
 
+    async def name_error():
+        "Raise a ``NameError``"
+        getattr(doggypants)
+
+
+    async def main():
+        """Test breakpoint in a streaming actor.
+        """
+        async with tractor.open_nursery(
+            debug_mode=True,
+            loglevel='error',
+        ) as n:
+
+            p0 = await n.start_actor('bp_forever', enable_modules=[__name__])
+            p1 = await n.start_actor('name_error', enable_modules=[__name__])
+
+            # retreive results
+            stream = await p0.run(breakpoint_forever)
+            await p1.run(name_error)
+
+
+    if __name__ == '__main__':
+        trio.run(main)
+
+
+You can run this with::
+
+    >>> python examples/debugging/multi_daemon_subactors.py
+
+And, yes, there's a built-in crash handling mode B)
+
+We're hoping to add a respawn-from-repl system soon!
+
+
+Worker poolz are easy peasy
+---------------------------
+It seems the initial ask from most new users is "how do I make a worker
+pool thing?".
+
+``tractor`` is built to handle any SC process tree you can
+imagine; the "worker pool" pattern is a trivial special case.
+
+We have a `full re-implementation <concurrent_actors_primes>`_ of the std-lib's
+``concurrent.futures.ProcessPoolExecutor`` example for reference.
+
+You can run it like so (from this dir) to see the process tree in
+real time::
+
+    $TERM -e watch -n 0.1  "pstree -a $$" \
+        & python examples/parallelism/concurrent_actors_primes.py \
+        && kill $!
+
+This uses no extra threads, fancy semaphores or futures; all we need
+is ``tractor``'s IPC!
+
+
+.. _concurrent_actors_primes: https://github.com/goodboy/tractor/blob/master/examples/parallelism/concurrent_actors_primes.py
```

Comment:
@guilledk not sure why it's not working?

Reply:
This might be it:
https://sublime-and-sphinx-guide.readthedocs.io/en/latest/references.html#use-custom-link-text

> Do not include a space between the last word of the link text and the opening angle bracket for the anchor text.

Reply:
Lol nope nm.

Reply:
Yeah no idea. The code looks right -> seems to be some GH thing.